tests/hash-backend-performance/rollup.config.ts

```diff
@@ -4,15 +4,15 @@ import typescript from "@rollup/plugin-typescript";
 import type { RollupOptions } from "rollup";
 
 const PRODUCTION = !process.env.ROLLUP_WATCH;
-const OUT_DIR = "dist/cjs";
+const OUT_DIR = "dist/esm";
 
 const bundles: RollupOptions[] = [
   {
     input: "src/main.ts",
     output: {
       dir: OUT_DIR,
-      entryFileNames: "[name].cjs",
-      format: "cjs",
+      entryFileNames: "[name].mjs",
```

Comment:
Strictly speaking I don't think this is required, as `.js` files will automatically be interpreted as type `"module"` due to its presence in `package.json`, but not a problem to leave it as-is
```suggestion
      entryFileNames: "[name].js",
```

Reply:
Ahh, fair enough!